### ISSUE TITLE
requires_stress on older models

### ISSUE THREAD

**Issue by OMalenfantThuot** [atomistic-machine-learning/schnetpack#188]
Hi.
I do not know if it is intented but models trained before the addition of stress tensors can no longer be used in evaluation mode.

I think it could be easily fixed in `/src/schnetpack/atomistic/model.py`
with something like:
```
try:
    if self.requires_stress:
        ...
except AttributeError:
    pass
```

As always, thank you very much!

**Comment by Stefaanhess** [comment]
Hey,
I see the problem. I guess it would be good to also store the best model parameters as a state dict, so one can initialize the model and load the weights?

``
model = spk.AtomisticModel(*args, **kwargs)
model.load_state_dict(torch.load("best_model.state_dict"))
``

This should solve the issues with changes in the model classes (also for the future), but requires knowledge about the model arguments for initialization.

**Comment by OMalenfantThuot** [comment]
This would be put in the spk_run.py script?

At the moment, the Error is in the `forward()` call, not the `__init__()` so I'm not sure this would fix the problem.

**Comment by Stefaanhess** [comment]
I meant that just as a general idea for saving and loading models. I will talk to @mgastegger about a solution tomorrow.
For now I made a small script which should transform old models to the new spk version:

```
import schnetpack as spk
import torch

# location to old version model
model_path = "qm9_energy_U/best_model"

# load old model
saved_model = torch.load(
    model_path,
    map_location="cpu"
)

# set the stress argument
for om in saved_model.output_modules:
    om.stress = None

# initialize new atomistic model
updated_model = spk.AtomisticModel(
    representation=saved_model.representation,
    output_modules=saved_model.output_modules
)

# save updated model
torch.save(updated_model, model_path + ".updated")
```
This adds the stress arguments to all output modules and creates a new `AtomisticModel` with the new code. The weights should stay the same.

**Comment by mgastegger** [comment]
We added a custom load function `load_model` in schnetpack.utils which should resolve the issue.

**Comment by OMalenfantThuot** [comment]
Works great now.
Thank you very much.
